tests/e2e/.env.example

```
@@ -0,0 +1,24 @@
+# Copy to tests/e2e/.env for manual runs.
+# `bin/setup-docker.sh` writes this file for you against the Docker stack.
+# In CI these are provided by the GitHub Actions job environment.
+
+# Site under test (Docker stack default).
+BASE_URL=http://localhost:8888
+
+# WP admin credentials — used for the UI login + saved session, and (via the
+# WP-API Basic-Auth plugin) for the API project's HTTP Basic auth.
+WP_ADMIN_USER=admin
+WP_ADMIN_PASSWORD=password
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The WP_ADMIN_PASSWORD key should go before the WP_ADMIN_USER key

(UnorderedKey)